### PR TITLE
Connected genset helper relay

### PR DESCRIPTION
**Pull request by ReinvdZee**
Some connected gensets support readout via modbus, but cannot be started remotely over modbus.
They require a start relay to be started.
Add a new relay option to the gui.

https://github.com/victronenergy/venus/issues/1286

### CODE REVIEW

File: pages/settings/devicelist/ac-in/PageAcInModelGenset.qml [hunk 1]
```diff
@@ -36,14 +36,29 @@ ObjectModel {
 		uid: root.bindPrefix + "/NrOfPhases"
 	}
 
+	readonly property var gensetEnabled: VeQuickItem {
+		uid: root.startStopBindPrefix ? root.startStopBindPrefix + "/Enabled" : ""
+	}
+
+	ListLabel {
+		allowed: root.gensetEnabled.value === 0
+		//% "Configure helper relay \n\n"
```
Comment (chriadam) [inline pages/settings/devicelist/ac-in/PageAcInModelGenset.qml:45]: does Qt handle split context comments?  Might be best to just stick it on one line

File: pages/settings/devicelist/ac-in/PageAcInModelGenset.qml [hunk 2]
```diff
@@ -36,14 +36,29 @@ ObjectModel {
 		uid: root.bindPrefix + "/NrOfPhases"
 	}
 
+	readonly property var gensetEnabled: VeQuickItem {
```
Comment (DanielMcInnes) [inline pages/settings/devicelist/ac-in/PageAcInModelGenset.qml:39]: `readonly property var gensetEnabled` -> `readonly property VeQuickItem gensetEnabled`

**Review by DanielMcInnes — APPROVED** [review]
1 nit, then lgtm

File: pages/settings/devicelist/ac-in/PageAcInModelGenset.qml (hunk 1 above)
Comment (ReinvdZee) [reply pages/settings/devicelist/ac-in/PageAcInModelGenset.qml]: fixed

File: pages/settings/devicelist/ac-in/PageAcInModelGenset.qml (hunk 2 above)
Comment (ReinvdZee) [reply pages/settings/devicelist/ac-in/PageAcInModelGenset.qml]: Fixed

**Comment by blammit** [comment]
@ReinvdZee could you rebase on latest main instead of adding a merge commit?

**Pull request merged**